go/vt/mysqlctl/backup.go

```diff
@@ -74,25 +77,38 @@ var (
 	// only used at backup time. Then it is put in the manifest,
 	// and when decoding a backup, it is read from the manifest,
 	// and used as the transform hook name again.
-	backupStorageHook = flag.String("backup_storage_hook", "", "if set, we send the contents of the backup files through this hook.")
+	backupStorageHook string
 
 	// backupStorageCompress can be set to false to not use gzip
 	// on the backups. Usually would be set if a hook is used, and
 	// the hook compresses the data.
-	backupStorageCompress = flag.Bool("backup_storage_compress", true, "if set, the backup files will be compressed (default is true). Set to false for instance if a backup_storage_hook is specified and it compresses the data.")
+	backupStorageCompress = true
 
 	// backupCompressBlockSize is the splitting size for each
 	// compressed block
-	backupCompressBlockSize = flag.Int("backup_storage_block_size", 250000, "if backup_storage_compress is true, backup_storage_block_size sets the byte size for each block while compressing (default is 250000).")
+	backupCompressBlockSize = 250000
 
 	// backupCompressBlocks is the number of blocks that are processed
 	// once before the writer blocks
-	backupCompressBlocks = flag.Int("backup_storage_number_blocks", 2, "if backup_storage_compress is true, backup_storage_number_blocks sets the number of blocks that can be processed, at once, before the writer blocks, during compression (default is 2). It should be equal to the number of CPUs available for compression")
+	backupCompressBlocks = 2
 
 	backupDuration  = stats.NewGauge("backup_duration_seconds", "How long it took to complete the last backup operation (in seconds)")
 	restoreDuration = stats.NewGauge("restore_duration_seconds", "How long it took to complete the last restore operation (in seconds)")
 )
 
+func init() {
+	for _, cmd := range []string{"mysqlctl", "mysqlctld", "vtadmin", "vtbackup", "vtcombo", "vtctl", "vtctld", "vttablet", "vttestserver"} {
```

Comment:
i guarantee `vtadmin` does not need this flag. i know you did the `go list`, but that's just a preliminary list ... i think you need to trace through this one (which both I and @GuptaManan100 have had to do for others)